<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bathymetry</class>
 <widget class="QFrame" name="bathymetry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>1204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="Lbl_H1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;BATHYMETRY SETTINGS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_SelFiles">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bathymetry and SW solution files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QPlainTextEdit" name="PText_Files">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_UploadFiles">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select the bathymetric files to be used</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Upload bathymetry</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
     <item>
      <widget class="QPlainTextEdit" name="PText_Solution">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_UploadSolution">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Upload the shallow water solver solution files</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Select solution</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="VLY_Gravity">
     <item>
      <widget class="QComboBox" name="CmB_FileType">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select the format of the bathymetric files uploaded</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <item>
        <property name="text">
         <string>SimCenter format bathymetry</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoClaw (Type I) bathymetry</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoClaw (Type II) bathymetry</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoClaw (Type III) bathymetry</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AdCirc data</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CmB_SolFormat">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Select the format of the solutions files uploaded</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <item>
        <property name="text">
         <string>SimCenter solution format</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GeoClaw solution format</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AdCirc solution format</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HLY_Gravity">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QLabel" name="Lbl_Gravity">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Gravity direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CmB_Grav01">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Select if gravity is along the positive / negative axis</string>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CmB_Grav02">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Select the axis along with gravity acts</string>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="CmB_Library">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Presently SimLibrary is not available</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <item>
        <property name="text">
         <string>To be coming soon</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_Notice">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This option is not yet available. but watch out for updates that are coming soon.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="CmB_FlumeGeoType">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Select how flume geometry will be defined</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Provide coordinates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use standard OSU Flume</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Geometry from mesh</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_TopViewH2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Top view of flume&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_TopView">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/images/TopView_v2.png&quot; width=&quot;275&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_Breadth" stretch="2,1">
     <item>
      <widget class="QLabel" name="Lbl_Breadth">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Breadth of wave flume</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="DSpBx_Breadth">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Enter the breadth of the wave flume</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_Segments">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Segments along length &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="Tbl_Segments">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Table of segments along the length of the flume</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Z</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_AddSeg">
     <item>
      <widget class="QToolButton" name="Btn_AddSeg">
       <property name="toolTip">
        <string>Add segment along the length of the flume</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Add point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Btn_RemSeg">
       <property name="toolTip">
        <string>Remove segment along the length of the flume</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Remove point</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="VSp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>214</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
